operator/main.go

```diff
@@ -64,8 +65,10 @@ func main() {
 	flag.BoolVar(&enableGateway, "with-lokistack-gateway", false,
 		"Enables the manifest creation for the entire lokistack-gateway.")
 	flag.BoolVar(&enableGatewayRoute, "with-lokistack-gateway-route", false,
-		"Enables the usage of Route for the lokistack-gateway instead of Ingress (OCP Only!)")
-	flag.BoolVar(&enablePrometheusAlerts, "with-prometheus-alerts", false, "Enables prometheus alerts")
+		"Enables the usage of Route for the lokistack-gateway instead of Ingress (OCP Only!).")
+	flag.BoolVar(&enablePrometheusAlerts, "with-prometheus-alerts", false, "Enables prometheus alerts.")
+	flag.BoolVar(&enableGrafanaLabsStats, "with-grafana-labs-stats", false,
```

Comment:
Would you consider enabling this by default? Nobody ever opts in to this kind of thing and setting the default to false essentially means we'll never get stats from anyone using the operator :( 

It's really helpful to us to know how people are using Loki with this kind of functionality to be able to steer the direction of the project.

Reply:
Sure this came in by default typing to `false` accidentally.